xbmc/interfaces/json-rpc/AudioLibrary.cpp

```diff
@@ -1391,21 +1384,21 @@ JSONRPC_STATUS CAudioLibrary::RefreshAlbum(const std::string& method,
                                            const CVariant& parameterObject,
                                            CVariant& result)
 {
-  int albumID = (int)parameterObject["albumid"].asInteger();
-
   CMusicDatabase musicdatabase;
   if (!musicdatabase.Open())
     return InternalError;
 
-  //check if albumID is a valid one
+  // Check if albumID is a valid one
   CAlbum album;
+  const int albumID = static_cast<int>(parameterObject["albumid"].asInteger());
   if (!musicdatabase.GetAlbum(albumID, album, false))
     return InvalidParams;
 
-  std::string path = StringUtils::Format("musicdb://albums/{}/", albumID);
-
-  //execute the album refresh job
-  CMusicLibraryQueue::GetInstance().StartAlbumScan(path, true);
+  // Start rescraping additional information for the given album
+  const CVariant albumIdVariant{parameterObject["albumid"]};
```

Comment:
Move this up to line 1393 and use the variable also in line „const int albumID = …“